src/puya/awst_build/validation/inner_transactions.py

```diff
@@ -11,16 +11,36 @@
 
 logger = log.get_logger(__name__)
 
+INNER_TRANSACTION_ASSIGNMENT_EXPRESSION_ERROR = (
+    "Inner Transactions cannot be used in assignment expressions."
```

Comment:
probably want to lower case `Inner Transaction(s)` in these messages, and remove the full stop at the end for single sentences?